AutoGraphTests/Stub.swift

```diff
@@ -1,10 +1,36 @@
 import Foundation
-import OHHTTPStubs
+
```

Comment:
OHHTTPStubs requires Obj-C, so I had to write a new network stubbing system that was pure Swift which is most of the changes in this file.